scissors/scissors_game/scissors.c

```diff
@@ -36,14 +44,29 @@ int play_scissors(void)
 	char res [10];
 	memset (res, 0, 10);
 	scanf ("%1s", &input);
-	if (char_to_text(input, res, sizeof(res)-1)==-1)
+	int user = -1;
+    user = char_to_text(input, res, sizeof(res)-1);
```

Comment:
1. Broken indentation.
2. What's the purpose of separated variable definition and initialization in the middle of code block?
Please stick to one codestyle:
- either define all variables at the beginning of code block,
- or combine definition with assigning a value.